client/http_curl.cpp

```diff
@@ -210,6 +212,40 @@ int libcurl_debugfunction(
     return 0;
 }
 
+std::string get_http_version(int _enum) {
```

Comment:
According to the documentation ( https://curl.se/libcurl/c/CURLINFO_HTTP_VERSION.html ) valid values are:
- CURL_HTTP_VERSION_1_0
- CURL_HTTP_VERSION_1_1
- CURL_HTTP_VERSION_2_0
- CURL_HTTP_VERSION_3
- or 0 if the version cannot be determined.

See no reason to check for other values